packages/eslint-plugin-reselect-utils/src/utils/arePropsDifferent.ts

```diff
@@ -0,0 +1,38 @@
+import * as ts from 'typescript';
```

Comment:
may be we can use direct imports instead of asterisk import

Reply:
Fixed